src/boolean-util/boolean-util.ts

```diff
@@ -0,0 +1,14 @@
+export namespace BooleanUtil {
+  export function valueOf(boolStr: string): boolean {
+    const TRUE_REGEXP = /^(t(rue)?|y(es)?|on|1)$/i;
+    const FALSE_REGEXP = /^(f(alse)?|n(o)?|off|0)$/i;
+
+    if (TRUE_REGEXP.test(boolStr.toLowerCase())) {
+      return true;
+    }
+    if (FALSE_REGEXP.test(boolStr.toLowerCase())) {
+      return false;
+    }
+    throw new Error(`unable to convert "${boolStr}" to boolean type`);
```

Comment:
가급적 함수의 마지막은 return이 되도록 해주세요.
```suggestion
    if (!FALSE_REGEXP.test(boolStr.toLowerCase())) {
      throw new Error(`unable to convert "${boolStr}" to boolean type`);
    }
    return false;
```

Reply:
수정했습니다!